src/Data/API/JSONToCBOR.hs

```diff
@@ -99,7 +104,18 @@ jsonToCBORBasic bt v = case (bt, v) of
     (BTbool  , _)        -> error "serialiseJSONWithSchema: expected bool"
     (BTint   , Number n) | Right i <- (floatingOrInteger n :: Either Double Int) -> TInt i
     (BTint   , _)        -> error "serialiseJSONWithSchema: expected integer"
-    (BTutc   , String t) -> TTagged 0 (TString t)
+    (BTutc   , String t) ->
+      TTagged 1000 (TMap [ (TInt 1, TInt secs)
+                         , (TInt (-12), TInt psecs) ])
+        where  -- taken from @Codec.Serialise.Class@:
```

Comment:
It's a shame we have to duplicate this logic, but I think it's okay for the time being. It might be worth exploring going to `Encoding` rather than `Term` in these functions, which should allow us to reuse `encode` for `UTCTIme`, but that's probably a bit big a change to make right now.